worker/lib/helpers/index.ts

```diff
@@ -127,7 +127,13 @@ export function getIpFromIface(iface: string): string {
 	throw new Error(`Could not find connected interface ${iface}`);
 }
 
+var resolverCache: { [target: string]: string } = {};
```

Comment:
Should we use a real cache here with limits, etc? Like memoize?

I guess since this is the core service the lifetime of the app should be short so maybe it doesn't matter.

Reply:
I think bringing in a memoization module wouldn't be bad if/when we have more than one use case. My immediate thoughts are that this probably doesn't need to be any more complicated than it is, but I'm open to other feedback.

Reply:
The `target` in this case is the short uuid.local, correct?

Reply:
@klutchell Correct

Reply:
@klutchell It seems lodash includes a `memoize` function, and we already have this as a dependency for the worker, so I'll rework this real quick to use that.